tests/test_Pretrain.py

```diff
@@ -27,13 +17,13 @@
 }
 
 config2_dict = {
-  "use_theano": True,
+  "use_tensorflow": True,
```

Comment:
Why is this needed at all? Does it actually init the engine?

Reply:
No, it does not init the engine. But it does call TF code, see other comment.

Reply:
I think we should change that. I don't really see a good reason why it really needs TF.